File: 2572-append-characters-to-string-to-make-subsequence/append-characters-to-string-to-make-subsequence.cpp
class Solution {
public:
    int appendCharacters(string s, string t) {
        int cnt=0;
        int i=0,j=0;
        while(i<s.size() && j<t.size())
        {
            if(s[i]==t[j])
            {
                cnt+=1;
                i+=1;
                j+=1;
            }
            else
            {
                i+=1;
            }
        }
        int res=t.size()-cnt;
        return res;
    }
};

File: 2572-append-characters-to-string-to-make-subsequence/append-characters-to-string-to-make-subsequence.py
class Solution:
    def appendCharacters(self, s: str, t: str) -> int:
        c=0
        i,j=0,0
        while(i<len(s) and j<len(t)):
            if s[i]==t[j]:
                c+=1
                i+=1
                j+=1
            else:
                i+=1
        res = len(t)-c
        return res